mods/ts/rules/trees.yaml

```diff
@@ -113,3 +125,78 @@ VEINHOLE:
 		Palette: player
 	WithSpriteBody:
 	AutoSelectionSize:
+
+FONA01:
+	Inherits: ^Tree
+	Tooltip:
+		Name: Fona
```

Comment:
Though "fauna" would still be at least questionable, since it refers to the animal part of "flora and fauna".